backend/src/main/java/de/neuefische/capstone/backend/entries/EntriesService.java

```diff
@@ -9,11 +10,26 @@ public class EntriesService {
 
     private final EntriesRepo entriesRepo;
 
-    public EntriesService(EntriesRepo entriesRepo) {
+    private final IdService idService;
+
+    public EntriesService(EntriesRepo entriesRepo, IdService idService) {
+        this.idService = idService;
         this.entriesRepo = entriesRepo;
     }
 
     public List<Entry> getAllEntries() {
         return entriesRepo.findAll();
     }
+
+    public Entry addEntry(EntryWithNoId entryWithNoId) {
+        return entriesRepo.save(new Entry(
+                idService.createRandomId(),
+                entryWithNoId.getTitle(),
+                entryWithNoId.getDescription(),
+                entryWithNoId.getDate(),
+                entryWithNoId.getAmount(),
+                entryWithNoId.getCategory(),
+                entryWithNoId.getInterval()
+        ));
+    }
```

Comment:
just out of curiosity why did you use the save and not insert method here?

Reply:
Thats a really good point. I should use insert here. Thanks:D